### PR TITLE
*: use yarn instead of npm

### PR DESCRIPTION
**Pull request by such**
Open for discussion but local results are pretty encouraging in terms of speed and travis appears to be very good at caching yarn.

Local tests:
- With npm:
```
lerna clean --yes && npm cache clean && rm -Rf node_modules && time sh -c "npm install && lerna bootstrap --npm-client npm"
...
sh -c "npm install && lerna bootstrap --npm-client npm"  205,43s user 52,39s system 90% cpu 4:45,21 total
```

- With yarn:
```
lerna clean --yes && npm cache clean && rm -Rf node_modules && time sh -c "yarn && lerna bootstrap --npm-client npm"
...
sh -c "yarn && lerna bootstrap --npm-client yarn"  110,49s user 45,26s system 132% cpu 1:57,96 total
```

`yarn.lock` files also fill a gap in npm 3 (but lock files are a feature of npm 4)

### CODE REVIEW

**Comment by t-bast** [comment]
Can you also update the READMEs to use "yarn install" instead of "npm install"?

**Comment by such** [comment]
good point

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ To get started, visit the [Indigo Framework website](https://indigoframework.com
 ## Development
 
 This multi-package repository (monorepo) is managed by [Lerna](https://github.com/lerna/lerna) to streamline development and testing of cross libraries changes. Make sure you've read Lerna documentation before starting.
+Lerna is configured to use [`yarn`](https://yarnpkg.com/en/) for dependency management. Make sure you have it up and running.
 
 Install dependencies:
 
 ```
-npm install
+yarn
```
Comment (t-bast) [inline README.md]: shouldn't this be "yarn install" ?

Comment (such) [reply README.md]: both are equivalent. I don't know if it's best to document the most explicit or the easiest one...

Comment (t-bast) [reply README.md]: Interesting, I didn't know that. Looks good then!

**Pull request merged**